### PR TITLE
RC78: Model collision shape property fix

### PR DESCRIPTION
**Pull request by dback2**
https://highfidelity.fogbugz.com/f/cases/20735

Pre-merge test plan:
- Enter Interface in desktop mode into any domain with edit permissions
- Open Create and select or create any Model entity
- Verify you can change the model's Collision Shape and Compound Shape (for compound shape setting, a URL to an OBJ file) properties as expected
- Exit and re-enter Create, return to the same model entity from above, verify the Collision Shape and Compound Shape properties are the same as before
- Select or create any Zone entity
- Verify you can change the zone's Shape Type and Compound Shape URL (for use compound shape URL setting, a URL to an OBJ file) properties as expected
- Also verify tooltips appear for Shape Type and Compound Shape URL properties as expected
- Exit and re-enter Create, return to the same zone entity from above, verify the Shape Type and Compound Shape URL properties are the same as before

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16571/HighFidelity-Beta-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16571/HighFidelity-Beta-Interface-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16571/HighFidelity-Beta-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16571/HighFidelity-Beta-Interface-PR14743-71aa392.exe

**Comment by dback2** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16576/HighFidelity-Beta-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16576/HighFidelity-Beta-Interface-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16576/HighFidelity-Beta-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16576/HighFidelity-Beta-Interface-PR14743-71aa392.exe

**Comment by dback2** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16580/HighFidelity-Beta-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16580/HighFidelity-Beta-Interface-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16580/HighFidelity-Beta-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16580/HighFidelity-Beta-Interface-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16583/HighFidelity-Beta-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16583/HighFidelity-Beta-Interface-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16583/HighFidelity-Beta-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16583/HighFidelity-Beta-Interface-PR14743-71aa392.exe

**Comment by qligmaloney** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16584/HighFidelity-Beta-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16584/HighFidelity-Beta-Interface-PR14743-71aa392.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16584/HighFidelity-Beta-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16584/HighFidelity-Beta-Interface-PR14743-71aa392.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16588/HighFidelity-Beta-PR14743-67c70b9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16588/HighFidelity-Beta-Interface-PR14743-67c70b9.exe

File: scripts/system/assets/data/createAppTooltips.json [hunk 1]
```diff
@@ -126,6 +126,14 @@
     "bloom.bloomSize": {
         "tooltip": "The radius of bloom. The higher the value, the larger the bloom."
     },
+    "zoneShapeType": {
+        "tooltip": "The shape of the zone's geometry.",
+        "jsPropertyName": "shapeType"
+    },
+    "zoneCompoundShapeURL": {
+        "tooltip": "The OBJ file to use for the compound shape if Shape Type is \"Use Compound Shape URL\".",
```
Comment (thoys) [inline scripts/system/assets/data/createAppTooltips.json]: This could be an FBX too. Perhaps change it to model file, or compound model file.

**Comment by thoys** [comment]
Build this please

**Comment by jakram** [comment]
Testing (when built)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16600/HighFidelity-Beta-PR14743-67c70b9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16600/HighFidelity-Beta-Interface-PR14743-67c70b9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16600/HighFidelity-Beta-PR14743-67c70b9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16600/HighFidelity-Beta-Interface-PR14743-67c70b9.exe

**Comment by conklin94122** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16634/HighFidelity-Beta-PR14743-67c70b9.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16634/HighFidelity-Beta-PR14743-67c70b9.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16634/HighFidelity-Beta-Interface-PR14743-67c70b9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16634/HighFidelity-Beta-PR14743-67c70b9.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16634/HighFidelity-Beta-Interface-PR14743-67c70b9.exe

**Pull request merged**